packages/content-type-read-receipt/src/ReadReceipt.ts

```diff
@@ -8,36 +8,25 @@ export const ContentTypeReadReceipt = new ContentTypeId({
   versionMinor: 0,
 });
 
-export type ReadReceipt = {
-  /**
-   * The timestamp the read receipt was sent, in ISO 8601 format
-   */
-  timestamp: string;
-};
-
-export type ReadReceiptParameters = Pick<ReadReceipt, "timestamp">;
+export type ReadReceipt = object;
```

Comment:
`object` is not a good type as it will match mostly anything that's not a primitive. i'm not sure what we should set this to since there's technically no content.

Reply:
The linter fails when I pass an empty {} but we could override the linter?

Reply:
`{}` = `any` 😭. an empty object is `Record<string, never>`.

Reply:
ahh got it so should we just pass that? 
```suggestion
export type ReadReceipt = Record<string, never>;
```

Reply:
yep, that should do the trick!